drivers/homeless_summary_report/app/views/homeless_summary_report/warehouse_reports/reports/details.haml

```diff
@@ -35,4 +35,6 @@
               %td= client.first_name
               %td= client.last_name
               - @data_cells.each do |cell|
-                %td= client.send("spm_#{cell}")
+                - val = client.send("spm_#{cell}")
+                - val = nil if val.negative? # -1 days is used to identify clients with no return
```

Comment:
I think we should specifically check the cell since it's possible we'd get negative numbers for other reasons (though maybe not currently).